### ISSUE TITLE
Excessive artist metadata when scrobbling from Qobuz (via BubbleUPnP Android app)

### ISSUE THREAD

**Issue by lbristol** [Saiyato/volumio-lastfm-plugin#18]
Hello,

I'm running Volumio 2 on a Raspberry Pi 4. I'm using the BubbleUPnP Android app to stream from my Qobuz account to the device, and I'm using this plugin for Last.fm scrobbling. 

Due to a duplicate scrobble issue, I disconnected Last.fm from the native Qobuz web app. Scrobbling now works, i.e. currently playing tracks are shown in Last.fm and data is being recorded for single, individual scrobbles, but those data include way too much artist information. For instance, instead of:

`The Who`

I'm getting:

`The Who, MainArtist - Pete Townshend, ComposerLyricist - Kit Lambert, Producer`

![image](https://user-images.githubusercontent.com/15706868/106010222-328ec180-6087-11eb-9c9c-e065f6d45e7e.png)

This is making it so that scrobbles aren't lining up with Last.fm track information. I could edit each scrobble to match up with Last.fm's metadata (album art, etc.) but that defeats the purpose of scrobbling in the first place - any advice? 

**TL;DR** I'm using BubbleUPnP to avoid paying for MyVolumio when streaming Qobuz through Volumio - but too much metadata is being sent to Last.fm when using this plugin. Is there any way to reduce artist information to just the "MainArtist" portion of Qobuz's metadata?

**Comment by Saiyato** [comment]
Hi,

Apologies for the late reply, lots of plugins and not so lots of free time :wink:
Can you post back what Volumio logs in "verbose: STATE SERVICE"?

It should look like this:
```
Jan 21 18:59:42 volumiotest volumio[25375]: verbose: STATE SERVICE {"status":"play","position":0,"seek":0,"duration":394,"samplerate":"88.2 kHz","bitdepth":"24 bit","channels":2,"random":false,"updatedb":false,"repeat":false,"bitrate":null,"isStreaming":false,"title":"Your Latest Trick","artist":"Dire Straits","album":"Brothers In Arms","uri":"NAS/HD/Dire Straits - Brothers in Arms/04 Your Latest Trick.flac","trackType":"flac"}

```
And formatted like this:
```
{
	"status": "play",
	"position": 0,
	"seek": 0,
	"duration": 394,
	"samplerate": "88.2 kHz",
	"bitdepth": "24 bit",
	"channels": 2,
	"random": false,
	"updatedb": false,
	"repeat": false,
	"bitrate": null,
	"isStreaming": false,
	"title": "Your Latest Trick",
	"artist": "Dire Straits",
	"album": "Brothers In Arms",
	"uri": "NAS/HD/Dire Straits - Brothers in Arms/04 Your Latest Trick.flac",
	"trackType": "flac"
}
```
It typically uses the artist and album properties to send to LastFM for correction first, it then uses the corrected values. If you enable debug logging, the plugin logs something like this:

```
Jan 21 18:59:43 volumiotest volumio[25375]: info: [LastFM] track info: {"success":true,"trackInfo":{"name":"Your Latest Trick","mbid":"6da22558-0a3d-4095-bbe4-e103907a33a5","url":"https://www.last.fm/music/Dire+Straits...
```
However only the album title is used to update the information for the plugin.
